cantools/database/__init__.py

```diff
@@ -78,6 +78,7 @@ def _load_file_cache(filename: StringPathLike,
                      frame_id_mask: Optional[int],
                      strict: bool,
                      cache_dir: str,
+                     sort_signals : bool,
```

Comment:
possibly, it's better to specify a compare function (or `None` if the signals ought not to be sorted). Anyway I'm also fine with the proposed solution...

Reply:
good idea, I have implemented that in commit a802630078421a75fd12d9ce9a931bc2c81f2379. And while implementing that I have found another bug, see commit bdc543c79af2e263f3f3bb906862b584bd5876ce. While I was at it I have also added a few more type hints in commit fedc1d7dc6e1a500c938d3770b5df43926ade892.